README.md

```diff
@@ -235,6 +235,137 @@ If you don't see a command prompt, try pressing enter.
 / # wget -O - <NGINX_POD_IP>
 ```
 
+# Testing
+
+## E2E testing
+
+E2E testing purpose is to validate submariner behaviour from an integration point of
+view. It needs to be executed in connection to an existing set of clusters.
+
+Config needs to be provided via a single kubeconfig file, you can squash it
+following the next command:
+
+  ```bash
+  export KUBECONFIG=creds/cluster1:creds/cluster2:creds/cluster3
+  kubectl config view --merge=true --flatten > creds/clusters-123
+  export KUBECONFIG=creds/clusters-123
```

Comment:
So you will have that kubeconfig by default for the tests (it's an alternative to specifying -kubeconfig on the cmdline). Or for using kubectl with the new config.

I will remove it to avoid confusion.